certified-connectors/SmartGlobalGovernance/apiDefinition.swagger.json

```diff
@@ -0,0 +1,992 @@
+				"responses": {
+					"200": {
+						"description": "The Record of Processing Activity was updated successfully"
+					},
+					"401": {
+						"description": "Authentication failed"
+					},
+					"404": {
+						"description": "Record of Processing Activity ID not found"
+					}
+				}
+			}
+		}
+	},
+	"x-ms-connector-metadata": [{
+			"propertyName": "Website",
+			"propertyValue": "https://www.smartglobalgovernance.com"
+		}, {
+			"propertyName": "Privacy policy",
+			"propertyValue": "https://www.smartglobalgovernance.com/fr/politique-confidentialite"
+		}, {
+			"propertyName": "Categories",
+			"propertyValue": "Data; Security"
+		}
+	],
+	"definitions": {
+		"Errors": {
+			"type": "array",
+			"items": {
+				"type": "string"
+			}
+		},
+		"Status": {
+			"type": "string"
+		},
+		"Client": {
+			"type": "object",
+			"properties": {
+				"client_id": {
+					"type": "integer",
+					"format": "int32",
+					"title": "Client ID"
+				},
+				"name": {
+					"type": "string",
+					"title": "Name"
+				},
+				"address": {
+					"type": "string",
+					"title": "Address"
+				},
+				"zip": {
+					"type": "string",
+					"title": "Zip"
+				},
+				"city": {
+					"type": "string",
+					"title": "City"
+				},
+				"country_iso": {
+					"type": "string",
+					"title": "Country ISO"
+				},
+				"contact_name": {
+					"type": "string",
+					"title": "Contact Name"
+				},
+				"contact_firstname": {
+					"type": "string",
+					"title": "Contact Firstname"
+				},
+				"contact_email": {
+					"type": "string",
+					"title": "Contact Email"
+				},
+				"contact_phone": {
+					"type": "string",
+					"title": "Contact Phone"
+				},
+				"complaint": {
+					"type": "integer",
+					"format": "int32",
+					"title": "Compliant"
+				},
+				"due_date": {
+					"type": "integer",
+					"format": "int32",
+					"title": "Due Date"
+				},
+				"agreements": {
+					"type": "integer",
+					"format": "int32",
+					"title": "Agreements"
+				},
+				"comments": {
+					"type": "string",
+					"title": "Comments"
+				}
+			}
+		},
+		"Processor": {
+			"type": "object",
+			"properties": {
+				"processor_id": {
+					"type": "integer",
+					"format": "int32",
+					"title": "Processor ID"
+				},
+				"name": {
+					"type": "string",
+					"title": "Name"
+				},
+				"address": {
+					"type": "string",
+					"title": "Address"
+				},
+				"zip": {
+					"type": "string",
+					"title": "Zip"
+				},
+				"city": {
+					"type": "string",
+					"title": "City"
+				},
+				"country_iso": {
+					"type": "string",
+					"title": "Country ISO"
+				},
+				"contact_name": {
+					"type": "string",
+					"title": "Contact Name"
+				},
+				"contact_firstname": {
+					"type": "string",
+					"title": "Contact Firstname"
+				},
+				"contact_email": {
+					"type": "string",
+					"title": "Contact Email"
+				},
+				"contact_phone": {
+					"type": "string",
+					"title": "Contact Phone"
+				},
+				"complaint": {

```

Comment:
@Philippe-Tivolle As per the swagger submitted in the certification portal, It is `compliant` ?

Reply:
There is a mismatch between the property that is exposed to the User (compliant) and the property that is expected by the API (complaint). It is a silly switch of letters but fixing it would require a developer and delay our submission.